src/select/__tests__/select-searchable-form-control.scenario.js

```diff
@@ -48,6 +50,32 @@ export default function Scenario() {
           onChange={params => setSearchable(params.value)}
         />
       </FormControl>
+
+      <label htmlFor="not-searchable-native-label">
+        Native not searchable
+        <Select
+          id="not-searchable-native-label"
+          clearable={false}
+          searchable={false}
+          placeholder="Select color"
+          value={nativeNotSearchable}
+          options={options}
+          onChange={params => setNativeNotSearchable(params.value)}
+        />
+      </label>
+
+      <label htmlFor="searchable-native-label">
+        Native searchable
+        <Select
+          id="searchable-native-label"
```

Comment:
Could we include a test that does not specify `id` and `htmlFor` attributes? One of the benefits of wrapping form elements in `<label>` is that the browser automatically wires up the connection. I realize that's not going to work for every complex form component, but it's a valid option for `<Select>` after the change in #4402.